ForumProject/users/views.py

```diff
@@ -1,11 +1,28 @@
-from rest_framework_simplejwt.views import (TokenObtainPairView as BaseTokenObtainPairView,
-                                            TokenRefreshView as BaseTokenRefreshView)
-
+from rest_framework_simplejwt.views import (
+    TokenObtainPairView as BaseTokenObtainPairView,
+    TokenRefreshView as BaseTokenRefreshView,
+)
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from rest_framework_simplejwt.tokens import RefreshToken
+from rest_framework.permissions import IsAuthenticated
 
 class TokenObtainPairView(BaseTokenObtainPairView):
     throttle_scope = 'token_obtain'
 
-
 class TokenRefreshView(BaseTokenRefreshView):
     throttle_scope = 'token_refresh'
 
+class LogoutView(APIView):
+    permission_classes = [IsAuthenticated]
+
+    def post(self, request):
+        try:
+            refresh_token = request.data["refresh_token"]
+            token = RefreshToken(refresh_token)
+            token.blacklist()
+            return Response({"message": "User successfully logged out."}, status=status.HTTP_200_OK)
+        except Exception as e:
+            return Response({"error": "Invalid token."}, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
Specific exception handling could improve error granularity. For example, handling `TokenError` separately could provide a specific message for token-related issues, while other unexpected issues could be logged or handled differently